### PR TITLE
Removed local /containers/lxd page

### LINKED ISSUE #3282: beta updates: kill product pages maas, lxd & juju etc
add redirects

- /juju to jujucharms.com
- /server-provisioning to maas.io
- /lxd to linuxcontainers.org

also update the links in the relevant pages on the site like /server has links to all three of these

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- removed from the footer nav - navigation.yaml
- updated all the redirects to linuxcontainers.org - redirects.yaml
- updated links in
  - /containers/docker-ubuntu
  - /containers
  - /server
- deleted the page - templates/containers/lxd.html
- archived the [copy doc]()

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/containers](http://0.0.0.0:8001/containers) - see that lxd page goes to linuxcontainers.org and the footer and menu no longer have lxd in it 
- go to the [lxd](http://0.0.0.0:8001/containers/lxd) page and see that you go to linuxcontainers.org

## Issue / Card

Partly fixes #3282

### CODE REVIEW

File: templates/server/index.html [hunk 1]
```diff
@@ -187,7 +187,7 @@ <h3 class="p-heading-icon__title">MAAS &mdash; bare metal provisioning</h3>
           <h3 class="p-heading-icon__title">LXD &mdash; system containers</h3>
         </div>
         <p>LXD, the Linux container hypervisor, merges the speed and density of containers with the manageability and security of traditional virtual machines. Economics are directly tied to density, and no other virtualisation technology is as fast or dense as LXD.</p>
-        <p><a href="/containers/lxd">Learn more about LXD&nbsp;&rsaquo;</a></p>
+        <p><a class="p-link--external" href="https://linuxcontainers.org/">Learn more about LXD&nbsp;&rsaquo;</a></p>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/server/index.html]: doh

[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/server/index.html]: @anthonydillon ready for re-review

**Comment by anthonydillon** [comment]
@pmahnke LGTM but you have a conflict

**Comment by pmahnke** [comment]
closed and replaced

**Pull request closed without merging**